src/content/configuration/module.mdx

```diff
@@ -248,6 +249,105 @@ module.exports = {
 }
 ```
 
+If `Rule.type` is an `asset` then `Rules.parser` option may be an object or function that describes a condition whether to encode file contents to Base64 or emit it as a separate file info the output directory
```

Comment:
I don't understand the last part, you mean: "or emit it as a separate file info to the output directory"?

Reply:
ooops.... fixed )